Add note for updater app (#815)
owncloud/docs-server: modules/admin_manual/pages/installation/manual_installation/script_guided_install.adoc

== General Info

It is important to understand how these scripts work and why things are done to process all the steps for a successful, smooth and secure installation or upgrade.

> IMPORTANT: *Do not use* this script in conjunction with the xref:maintenance/upgrading/update.adoc[Upgrading ownCloud with the Updater App]. When using the Updater App, your webserver must have write access to core files to overwrite them. Therefore it is needed to set much weaker permissions when using the Updater App which can be in some circumstances a security issue. *The scripts used with the _Script-Guided Installation_ use strong permissions to prevent the webserver overwriting core files.* Only defined locations handled by the script are allowed for the webserver to write into. This greatly increases security and hardens the environment ownCloud Server is running.

Reviewer: ```suggestion
IMPORTANT: *Do not use* this script in conjunction with xref:maintenance/upgrading/update.adoc[Upgrading ownCloud with the Updater App]. When using the Updater App, your web server must have write access to core files to overwrite them. Therefore it is necessary to set much weaker permissions when using the Updater app which can be in some circumstances a security issue. *The scripts used with the _Script-Guided Installation_ use strong permissions to prevent the web server overwriting core files.* The web server can only write to defined locations handled by the script. This greatly increases security and hardens the environment of the ownCloud server.
```
